source/MaterialXGenGlsl/GlslShaderGenerator.cpp

```diff
@@ -258,33 +258,33 @@ ShaderPtr GlslShaderGenerator::generate(const string& shaderName, ElementPtr ele
     shader.addLine("#version " + getVersion(), false);
     shader.newLine();
 
+    // Add all private constants
+    const Shader::VariableBlock& vsPrivateConstants = shader.getConstantBlock(HwShader::VERTEX_STAGE);
+    if (!vsPrivateConstants.empty())
+    {
+        shader.addComment("Constant block: " + vsPrivateConstants.name);
+        emitVariableBlock(vsPrivateConstants, true, shader);
+    }
```

Comment:
Maybe remove "private" and just call them vsConstants, since there is only a single block of constants now.